regolith/schemas.py

```diff
@@ -86,18 +86,17 @@
     "contacts": {
         "_id": "afriend",
         "aka": [
-            "A. B. Friend"
-            "AB Friend"
+            "A. B. Friend",
+            "AB Friend",
             "Tony Friend"
         ],
         "department": "physics",
         "email": "friend@deed.com",
         "institution": "columbiau",
         "name": "Anthony B Friend",
-        "notes": "The guy I meet for coffee sometimes",
+        "notes": ["The guy I meet for coffee sometimes"],
```

Comment:
check whether this can be a string or a list in the schema.   If it can be either we need to modify the code to handle either.

Reply:
up there!